rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -152,13 +173,13 @@ void setUp() {
         }
 
         @Test
-        void processValidationFailure_whenSvpFundTxHashUnsigned_shouldLogValidationFailureAndClearValue() {
+        void updateSvpState_whenSvpFundTxHashUnsigned_shouldLogValidationFailureAndClearValue() {
             // arrange
-            Sha256Hash svpFundTxHashUnsigned = BitcoinTestUtils.createHash(1);
-            bridgeStorageProvider.setSvpFundTxHashUnsigned(svpFundTxHashUnsigned);
+            svpFundTransactionHashUnsigned = BitcoinTestUtils.createHash(1);
+            bridgeStorageProvider.setSvpFundTxHashUnsigned(svpFundTransactionHashUnsigned);
 
             // act
-            bridgeSupport.processSVPFailure(proposedFederation);
+            bridgeSupport.updateSvpState(rskTx);
```

Comment:
Would it be possible to test by calling `updateCollections` directly? Or does that require a complex setup?

It would make the test super complete and `updateSvpState` could be private